src/main/java/net/masterthought/cucumber/reducers/ReportFeatureMerger.java

```diff
@@ -0,0 +1,20 @@
+package net.masterthought.cucumber.reducers;
+
+import net.masterthought.cucumber.json.Feature;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+public interface ReportFeatureMerger extends Predicate<List<ReducingMethod>> {
```

Comment:
This change is quite big to review and understand - would you split it into smaller changes? 
The first one would be add this interface with default implementation so after that the report works as now but is ready for further modification

It should really speed up review and merging process